docs/source/code-generation/codegen-getting-started.mdx

```diff
@@ -0,0 +1,293 @@
+
+## Step 3: Move the SDL file into your project
+
+In the root directory of the `apollo-ios` clone, add a directory called `codegen-test` and move the downloaded `star-wars-swapi@current.graphql` file into this directory.
+
+Rename the file to `star-wars-swapi@current.graphqls`.
+
+
+## Step 4: Create a test GraphQL operations file
+
+You'll also need to create a basic operations file to complete the codgen process.
+
+We'll be using the [allFilms Query operation](https://studio.apollographql.com/public/star-wars-swapi/explorer?variant=current) as an example.
+
+Copy the entire query you see in Apollo Studio and create a new file called `star-wars-swapi@current.graphql` in the `codegen-test` directory and paste this query into the file.
+
+```graphql
+
+query Query {
+  allFilms {
+    films {
+      title
+      director
+      releaseDate
+      speciesConnection {
+        species {
+          name
+          classification
+          homeworld {
+            name
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+## Step 4: Compile the Codegen CLI
+
+In the root directory of the `apollo-ios` project there is a `makefile` which includes a `build-cli` rule that can be used to compile the CLI. From the root of your `apollo-ios` clone, in a Terminal run:
+
+```bash
+make build-cli
+```
+
+This will compile the Codegen CLI in release mode for you. The Codegen CLI executable will now be located in your `apollo-ios` clone at `.build/release/apollo-ios-cli`.
+
+
+## Step 5: Move the CLI executable
+
+Go to `.build/release/apollo-ios-cli` and move the `apollo-ios-cli` executable file to your `codegen-test` directory. This will put the CLI in the same directory as your schema and operation file for testing purposes.
+
+## Step 6: Initialize a new configuration
+
+Switch to the the `codegen-test` directory, and in a Terminal run:
+
+```bash
+./apollo-ios-cli init --schema-name TestSchema
+```
+
+This will execute the initialization process to create an `apollo-codegen-configuration.json` file with default values.  The file should look like this:
+
+```json
+{
+  "schemaName" : "TestSchema",
+  "input" : {
+    "operationSearchPaths" : [
+      "**/*.graphql"
+    ],
+    "schemaSearchPaths" : [
+      "**/*.graphqls"
+    ]
+  },
+  "output" : {
+    "testMocks" : {
+      "none" : {
+      }
+    },
+    "schemaTypes" : {
+      "path" : "./Root",
+      "moduleType" : {
+        "swiftPackageManager" : {
+        }
+      }
+    },
+    "operations" : {
+      "relative" : {
+      }
+    }
+  }
+}
+
+```
+
+### Step 7: Generate code
+
+Now you are ready to generate some code!  Within the `codegen-test` directory in a Terminal run,
+
+```bash
+./apollo-ios-cli generate --path apollo-codegen-config.json
+```
+
+you should see a new file created in the `codegen-test` directory called `Query.graphql.swift` that looks like this:
+
+<ExpansionPanel title="Swift Generated Code">
```

Comment:
```suggestion
<ExpansionPanel title="Generated Code">
```
I initially read it as Swift did the code generation. The type of code is captured in the tab anyways.
<img width="235" alt="Screen Shot 2022-11-09 at 1 08 27 PM" src="https://user-images.githubusercontent.com/146856/200942151-3c7065d0-c9ea-4796-9661-28f79d50d84d.png">